app/controllers/concerns/csv_email_action.rb

```diff
@@ -25,7 +25,7 @@ def queue_csv_report_email(report)
     yield_email_and_respond_for_report do |email|
       # TODO: in order to use ActiveJob's `deliver_later`, the report needs to be
       # specifically serializable, which many of our reports are not.
-      CsvReportMailer.delay.csv_report_email(email, report)
+      CsvReportMailer.with(to_address: email, report: report).delay.csv_report_email
```

Comment:
Is `delay` the old way of doing `deliver_later`? Can we update this to use `deliver_later` (I don't know whether they are equivalent).